lib/vrl/stdlib/src/parse_aws_alb_log.rs

```diff
@@ -499,5 +515,113 @@ mod tests {
                              user_agent: "curl/7.46.0"})),
             tdef: TypeDef::new().fallible().object::<&str, Kind>(inner_type_def()),
         }
+
+        eight {
```

Comment:
I know the other tests in this file use this naming convention as well, but I'd prefer it if we use more descriptive names to make it easier to find tests that validate a certain condition.

Reply:
> that validate a certain condition.

That will be hard since, at least the tests added here, are an example of real logs returned by AWS and as such they haven't been created to test a specific condition but are testing multiple unrelated things.

Reply:
Ah okay. I assumed, since they were added in this PR, that they all tested some variation of "quoted lists". So `quoted_lists_1` etc would suffice.

But it's a minor thing, so happy to leave it as-is 👍